docs/getting-started/getting-started-with-docker.rst

```diff
@@ -22,15 +24,143 @@ There is a good chance that you have heard of containerization. A container is e
+
+Using existing Docker images
+----------------------------
+Sometimes, the tool you want to run is already Dockerized. Perhaps you want to use that existing image for your own research, or maybe you want to base a pipeline off it. But how would you know that such an image exists? Where do you look for Docker images? And how do you know if you've found a good one?
+
+Container registries
+~~~~~~~~~~~~~~~~~~~~
+Docker images are usually shared on registries. `Quay.io <https://quay.io/>`__ and `Docker Hub <https://hub.docker.com/>`__ are examples of popular public registries, which anyone can browse online. `GitLab Container Registry <https://about.gitlab.com/blog/2016/05/23/gitlab-container-registry/>`__ on the other hand is a private registry, so it can't be easily browsed by outside users.
+
+Container registries usually show you the layers that make up a particular Docker image, what versions are available, and the username or organization that the Docker image is associated with. They also usually give you the command line text you need to use in order to download a particular image locally, which will allow you use to the image on your own machine.
+
+.. note:: Different registries have different limits on how often you can download images from the command line. If you are not logged in with the service you are using, these limits are usually based upon IP address, and may start blocking you from downloading if you are doing it too many times in a short period of time. As of our writing this, `Docker Hub in particular limits your IP to 100 downloads per six hours <https://docs.docker.com/docker-hub/download-rate-limit/>`__ if you are not logged in.
+
+Looking for an official image
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+A good place to start when looking for Docker images (or for one to base your own Docker image upon) are official images maintained by institutions. Docker Hub makes this easy by maintaining `a curated list of official images <https://hub.docker.com/search?image_filter=official&q=>`__, and `a curated list of verified images maintained by commerercial entities <https://hub.docker.com/search?q=&image_filter=store>`__. These images include the likes of `Ubuntu <https://hub.docker.com/_/ubuntu>`__, `golang <https://hub.docker.com/_/golang>`__, the `AWS CLI <https://hub.docker.com/r/amazon/aws-cli>`__, and even `Docker <https://hub.docker.com/_/docker>`__. Yes, you can run Docker in Docker!
+
+Quay.io does not maintain a list of official images like Docker Hub, but you can nonetheless find official images on there with a bit of searching, such as `CentOS <https://quay.io/repository/centos/centos?tab=info>`__. Because Quay is owned by Red Hat, it is a good place to find images based on Red Hat Linux distributions.
+
+Looking for an oft-maintained image
```

Comment:
Do we want to add a section about container security? Maybe pull a section from the best practices for secure and fair workflows?